easybuild/easyconfigs/t/TOBIAS/TOBIAS-0.12.12-foss-2021b.eb

```diff
@@ -0,0 +1,76 @@
+easyblock = 'PythonBundle'
+
+name = 'TOBIAS'
+version = '0.12.12'
+
+homepage = 'https://github.com/loosolab/TOBIAS'
+description = """TOBIAS is a collection of command-line bioinformatics tools for
+performing footprinting analysis on ATAC-seq data, and includes: Correction of
+Tn5 insertion bias; Calculation of footprint scores within regulatory regions;
+Estimation of bound/unbound transcription factor binding sites; and
+Visualization of footprints within and across different conditions"""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('Pysam', '0.17.0'),
+    ('scikit-learn', '1.0.1'),
+    ('Seaborn', '0.11.2'),
+    ('PyYAML', '5.4.1'),
+    ('boto3', '1.20.13'),
+    ('pybedtools', '0.8.2'),
+    ('pyBigWig', '0.3.18'),
+    ('XlsxWriter', '3.0.2'),
+    ('adjustText', '0.7.3'),
+    ('Biopython', '1.79'),
+    ('ImageMagick', '7.1.0-4'),
+]
+
+exts_list = [
+    ('argparse', '1.4.0', {
```

Comment:
`argparse` is part of the python standard library in python3, is there a specific reason to have this (old) version here?

Reply:
It's either add argparse as a dependency, or `sed` it out of the requirements file, otherwise `pip check` complains. TOBIAS is written for Py2.7 (using six, so it should works even with 3.9), which I imagine is the reason.